packages/terra-action-header/src/terra-dev-site/doc/action-header/ActionHeader.1.doc.mdx

```diff
@@ -25,7 +38,7 @@ This component requires the following peer dependencies be installed in your app
 ## Implementation Notes:
 The Action-Header component must be composed inside the [Base][1] component with a locale in order for it to load the correct translation strings.
 
-[1]: https://github.com/cerner/terra-core/tree/master/packages/terra-base/docs
+[1]: https://github.com/cerner/terra-core/tree/master/packages/terra-base/src/terra-dev-site/doc/base/Base.1.doc.mdx
 
 ## Usage
```

Comment:
We should grab a few more opinions on this before making any changes, but I think we should consider removing the usage section all together or reducing it to include only the import during these mdx uplifts. The static code blocks can easily become outdated and the living code examples below provide a good example implementation and guarantee code integrity. 

This [PR](https://github.com/cerner/terra-core/pull/2781/files) for example reduced the usage block to include only the import.

Reply:
+1 to reducing to the import. The code blocks can show working code examples

Reply:
Updated here https://github.com/cerner/terra-core/pull/2802/commits/5339e5c49685a2febe7d1fd150600dc7892892f5.